.github/workflows/ci.yml

```diff
@@ -76,6 +77,16 @@ jobs:
         if: always()
         uses: ./.github/actions/upload-heapdumps
 
+      - name: Surefire reports
+        if:  (!cancelled() && contains(fromJSON(${{ env.RELEASE_BRANCHES }}), github.ref) && contains(fromJSON('["push", "workflow_dispatch"]'), github.event_name))
```

Comment:
Can you make the action handle this if statement instead of having to have it here?